### PR TITLE
SISRP-12127 - Change when Planner re-sizes to single term

### PR DESCRIPTION
**Pull request by darrylumich**
https://jira.berkeley.edu/browse/SISRP-12127

### CODE REVIEW

File: src/css/cs/items/_aa_planner.scss [hunk 1]
```diff
@@ -16,6 +16,10 @@
           display: none;
         }
       }
+      
+      .ps_box-longedit {
+        width: 300px;
```
Comment (christianvuerings) [inline src/css/cs/items/_aa_planner.scss:21]: @darrylumich fine with me, just make sure this works on smaller screens as well. Usually we use `max-width` instead.

**Comment by marcpechaitis** [comment]
@darrylumich we already have a "uc-control-width-###px" in sis-custom (for the range 105px > 320px in 5px increments)...would uc-control-width-300px  work instead and be less invasive?

**Comment by christianvuerings** [comment]
+1 on comment from @marcpechaitis

**Comment by marcpechaitis** [comment]
@christianvuerings & @darrylumich so did this change get backed out?  How do we do that?

**Comment by marcpechaitis** [comment]
@darrylumich and @christianvuerings nevermind I see that darryl removed it in pull #57

**Pull request merged**